### PR TITLE
Fix some basic JS CS

### PR DESCRIPTION
**Pull request by mvorisek**
no functional change

### CODE REVIEW

File: plugins/enigma/enigma.js [hunk 1]
```diff
@@ -199,16 +200,19 @@ rcube_webmail.prototype.enigma_key_create_save = function () {
     // validate the form
     if (!password || !confirm) {
         this.alert_dialog(this.get_label('enigma.formerror'));
+
```
Comment (alecpl) [inline plugins/enigma/enigma.js:203]: We didn't do this for PHO and I don't like it here too.

File: plugins/enigma/enigma.js [hunk 2]
```diff
@@ -304,28 +309,28 @@ rcube_webmail.prototype.enigma_export = function (selected) {
...
                 text: this.get_label('enigma.onlypubkeys'),
                 click: function (e) {
                     rcmail.enigma_export_submit(args);
                     $(this).remove();
-                }
+                },
             },
             {
-                'class': 'export',
+                class: 'export',
```
Comment (alecpl) [inline plugins/enigma/enigma.js:320]: I'm afraid that this will not work. 'class' is a keyword in javascript.

File: plugins/enigma/enigma.js [hunk 3]
```diff
@@ -532,7 +543,7 @@ rcube_webmail.prototype.enigma_add_list_row = function (r) {
 
 
 /*********************************************************/
-/*********        Enigma Message methods         *********/
+/* ********        Enigma Message methods        *********/
```
Comment (alecpl) [inline plugins/enigma/enigma.js:546]: This looks wrong.

Comment (mvorisek) [reply plugins/enigma/enigma.js:546]: There is a rule like in php, it should be fine.

Comment (alecpl) [reply plugins/enigma/enigma.js:539]: That added space does not make sense. Look at the line above (with all asterisks). If we can't fix it we might consider removing these whole comment blocks.

Comment (mvorisek) [reply plugins/enigma/enigma.js:539]: that is https://github.com/roundcube/roundcubemail/commit/f9b3ba60e3ba19c4153fdf46e1a85e4bc73541b0

IMO the comments can be helpful to place a new methods more logically as the files itself are quite large.

We did the same with PHP code, we simply made the fixer happy.

Thus I prefer to merge this PR as is.

**Pull request merged**